db/migrate/20150427224114_add_o_auth_to_app.rb

```diff
@@ -0,0 +1,9 @@
+class AddOAuthToApp < ActiveRecord::Migration
+  def change
+    add_column :apps, :redirect_uri, :string
+    add_column :apps, :homepage, :string
+    add_column :apps, :description, :string
+    add_column :apps, :privileged, :boolean, null: false, default: false
```

Comment:
I should note that the `privileged` field is there to specify that an app has been vetted by us and deemed "trustworthy enough" to give access to spooky-scary APIs like the "resource owner password" flow or the "global" scope.